File: Day_One/Javascript/NumberGuessingGame.js
const input = require("prompt-sync")();


let random_number = Math.floor(Math.random() * 100) + 1;

let attempt = 1;
console.log(random_number);
do{
const guess = Number(input("Guess the number between 1 and 100 => "));

if(guess == random_number){
    console.log("Congratulation, you got the number correct");
    break;
}

else if(guess < random_number){
    console.log("Too low");
}
else if(guess > random_number){
    console.log("Too High");
}

attempt += 1;

if(guess < 1 || guess > 100){
    console.log("Guess out of bound");
}


}while(attempt <= 5);

console.log("=====RESULTS=====");
console.log("Correct Number => "+ random_number);
console.log("Number of Attempts => "+ attempt);


if(attempt == 1){
    console.log("Lengendary");
}
if(attempt == 2){
    console.log("Excellent");
}
if(attempt == 3){
    console.log("Good");
}
if(attempt == 4){
    console.log("Good");
}

if(attempt == 5){
    console.log("Close");
}

if(attempt > 5){
    console.log("Better luck")
}

File: Day_One/python/number_guessing_game.py

import random 

random_number = random.randint(1, 100)
attempt = 1;

while attempt <= 5:
    try:
        guess = int(input("Guess the number between 1 and 100 => "))
        if guess ==random_number:
            print("congratulations you got the number")
            break
        elif guess  > random_number:
            print("Too high")
        elif guess < random_number:
            print("Too low")
        attempt += 1
        if guess > 100 or guess < 1:
            print("Out of bound")
    except ValueError:
        print("Please enter a valid number")


print("=====RESULTS=====")
print()
print()
print(f"Correct Number => {random_number}")
print()
print(f"Number of Attempts => {attempt}")
if attempt == 1:
    print("Legendary")

elif  attempt == 2:
    print("Excellent")

elif  attempt == 3 or attempt == 4:
    print("Good")

elif  attempt == 5:
    print("Close")

else:
    print("Better luck")


